project/ProjectPlugin.scala

```diff
@@ -83,37 +87,38 @@ object ProjectPlugin extends AutoPlugin {
         organizationEmail = "hello@47deg.com"
       ),
       orgLicenseSetting := ApacheLicense,
-      scalaVersion := "2.11.11",
+      scalaVersion := "2.12.10",
       scalaOrganization := "org.scala-lang",
-      crossScalaVersions := Seq("2.11.8"),
+      crossScalaVersions := Seq("2.12.10"),
```

Comment:
if we are going to just maintain the scala 2.12 version you don't need the crossCompile setting